packages/repo-tools/templates/typescript-backstage/modelGeneric.mustache

```diff
@@ -0,0 +1,36 @@
+{{#models}}
```

Comment:
Could we add a `{{! ` comment stating where these files are from? As well as a [notice](https://github.com/backstage/backstage/blob/master/NOTICE) that highlights the upstream licences that we're subject to?

Reply:
Should I update that top level notice or create a new template notice?

Reply:
yep the top-level one

Reply:
Added in df7450e78cb0561926881f6a0b06b3f66c1c06e9